.github/workflows/check_cirrus_cron.yml

```diff
@@ -13,5 +16,5 @@ on:
 jobs:
   # Ref: https://docs.github.com/en/actions/using-workflows/reusing-workflows
   call_cron_failures:
-    uses: containers/buildah/.github/workflows/check_cirrus_cron.yml@main
+    uses: containers/podman/.github/workflows/check_cirrus_cron.yml@main
```

Comment:
This is fine, but I'm just curious as to what prompted the change to Podman?  Are you centralizing all there?

Reply:
Yes exactly.  Buildah use to be the central-source, but it just reused what was in podman.  So this is simplifying that.